.env.example

```diff
@@ -15,6 +15,7 @@ WORKSPACE_NAME = 'aml-workspace'
 EXPERIMENT_NAME = ''
 
 # AML Compute Cluster Config
+AML_ENV_NAME='diabetes_regression__training_env'
```

Comment:
Double underscore in name. Is it for purpose?

Reply:
fixed